### PR TITLE
Added modern theme style to toast

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by Blackbaud-PaulCrowder** [comment]
Fixes #35

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-modern-theme)

File: src/app/public/modules/toast/toast.component.ts [hunk 1]
```diff
@@ -116,6 +123,12 @@ export class SkyToastComponent implements OnInit, OnDestroy {
     return classNames.join(' ');
   }
 
+  public baseIcon: SkyIconStackItem;
+
+  public topIcon: SkyIconStackItem;
+
+  public icon: string;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/toast/toast.component.ts:128]: Check your alphabetical ordering of properties.

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Nice work. Cross browser tests look good. Just one tiny style comment, then we can ship it.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-modern-theme](https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-modern-theme)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**